src/main/java/org/sagebionetworks/bridge/sdk/DeveloperClient.java

```diff
@@ -248,4 +250,39 @@
      * @return a list of upload schemas
      */
     public ResourceList<UploadSchema> getAllUploadSchemas();
+
+    /**
+     * Get all subpopulations defined for this study.
+     * @return
+     */
+    public ResourceList<Subpopulation> getAllSubpopulations();
+    
+    /**
+     * Create a new subpopulation.
+     * @param subpopulation
+     */
+    public GuidVersionHolder createSubpopulation(Subpopulation subpopulation);
+    
+    /**
+     * Get a subpopulation using its GUID.
+     * @param subpopGuid
+     * @return
+     */
+    public Subpopulation getSubpopulation(SubpopulationGuid subpopGuid);
+    
+    /**
+     * Update a subpopulation with the given GUID.
+     * @param subpopulation
+     */
+    public GuidVersionHolder updateSubpopulation(Subpopulation subpopulation);
+    
+    /**
+     * Delete a subpopulation. Studies start with a default subpopulation that cannot 
+     * be deleted (although it can be edited). But other subpopulations can be deleted. 
+     * Records of consent for a particular subpopulation are not deleted and historical 
+     * records can be produced of consent to deleted subpopulations.
+     * @param subpopGuid
+     */
+    public void deleteSubpopulation(SubpopulationGuid subpopGuid);
```

Comment:
If it fails to delete the subpopulation, it will throw an exception of some kind... 404 if there was no subpopulation, 500 if there was a server error.